roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```diff
@@ -0,0 +1,189 @@
+        tasks_from: check_user.yml
+      vars:
+        user: sr-cust+
+        expected_value: sr-cust+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: sr-custom
+        group: sr-custom-group
+    
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: schema-registry.log
+        custom_path: /kafka/logs/schemaregistry/
+        group: sr-custom-group
+        full_user: sr-custom
+        
+- name: Verify - Kafka Connect custom user and custom group and custom logging directory and log file presence
+  hosts: kafka_connect
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: connect+
+        expected_value: connect+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: connect-custom
+        group: connect-custom-group
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: connect-distributed.log
+        custom_path: /kafka/logs/kafkaconnect/
+        group: connect-custom-group
+        full_user: connect-custom
+
+- name: Verify - Rest Proxy custom user and custom group and custom logging directory and log file presence
+  hosts: kafka_rest
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: rest-cu+
+        expected_value: rest-cu+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: rest-custom
+        group: rest-custom-group
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: rest-proxy.log
+        custom_path: /kafka/logs/kafkarest/
+        group: rest-custom-group
+        full_user: rest-custom
+
+- name: Verify - KSQL custom user and custom Group and custom logging directory and log file presence
+  hosts: ksql
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: ksql-cu+
+        expected_value: ksql-cu+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: ksql-custom
+        group: ksql-custom-group
+        expected_value: ksql-custom-group
+      
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: ksql-streams.log
+        custom_path: /kafka/logs/ksql/
+        group: ksql-custom-group
+        full_user: ksql-custom
+
+- name: Verify - C3 custom user and custom Group and custom logging directory and log file presence
+  hosts: control_center
+  gather_facts: no
+  tasks:
+    - import_role:
```

Comment:
seems to me the check_user task does not need two arguments, since user and expected_value are the same everywhere

Reply:
maybe a better name as well, like "check_process_user.yml" makes more sense